Documentation/Supported Mods.md

```diff
@@ -51,6 +51,8 @@ To include supported mods in your multiworld slot, you need to include a section
 This section must be an array with the **exact** names of every mod you wish to include. Any improperly typed mod name will be ignored.
 ![image](https://i.imgur.com/uOHtXmU.png)
 
+If manually typing your mods list isn't working, try going to the [weighted settings page](https://archipelago.gg/weighted-options) and selecting stardew and then leaving all the other settings and check off the boxes for the mods you want. Export the settings, and copy paste the result into your own yaml file. 

```

Comment:
Could we try to type this in a more "formal" tone, like the rest of the document? "Try going to the weighted settings" feels like a shift in tone and less official than the surrounding stuff

Reply:
Sure! Let me do some thinking.

Reply:
does this sound good? 
<img width="884" alt="Screenshot 2024-01-29 at 6 30 55 PM" src="https://github.com/agilbert1412/StardewArchipelago/assets/78334662/1ce783eb-9b02-46b4-8dc7-63f0a2fbcb04">

Reply:
I just noticed a missing comma... oops